test/testSetup.ts

```diff
@@ -54,7 +54,11 @@ export class TestSetup {
 
   public async tearDown(): Promise<void> {
     await this.checkForUncaughtErrors();
-    await utilities.deleteScratchOrg(this.scratchOrgAliasName, this.reuseScratchOrg);
+    try {
+      await utilities.deleteScratchOrg(this.scratchOrgAliasName, this.reuseScratchOrg);
+    } catch (error) {
+      throw new Error(`Deleting scratch org failed with Error: ${(error as Error).message}`);
```

Comment:
@CristiCanizales why fail the test run because scratch org could not be deleted?

Reply:
copy and paste error 🤡 I meant utilities.log